presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -173,10 +173,15 @@ public AggregationBuilder source(PlanNode source)
 
         public AggregationBuilder addAggregation(Symbol output, Expression expression, List<Type> inputTypes)
         {
+            return addAggregation(output, expression, inputTypes, Optional.empty());
+        }
+
+        public AggregationBuilder addAggregation(Symbol output, Expression expression, List<Type> inputTypes, Optional<Symbol> mask)
```

Comment:
Maybe this method could get rid of `Optional`? As in case of `Optional.empty()` the method you just added can called.